net/remote/remote-client.sh

```diff
@@ -62,6 +65,7 @@ solana-bench-tps)
       --sustained \
       --threads $threadCount \
       $benchTpsExtraArgs \
+      -r ./client-accounts.yml \
```

Comment:
nit: just use the long form, it's self documenting!

Reply:
👍